velox/expression/tests/ExprTest.cpp

```diff
@@ -4210,6 +4215,44 @@ TEST_F(ExprTest, coalesceRowInputTypesAreTheSame) {
           e.message());
     }
   }
+
+  {
+    // cast(concat(c0) as row(row_field0)).row_field0 + coalesce(c1,
+    // concat(c0)).row_field0.
+    core::TypedExprPtr concat = std::make_shared<const core::ConcatTypedExpr>(
```

Comment:
whats preventing us from using an expression string here instead of creating the expression tree manually?

Reply:
I wanted to make sure that the result type of the first concat is Row<"":bigint> while the result type of the second concat is Row<"row_field0":bigint>. I'm not sure whether I can do this with the existing interface instead of creating the expression tree manually.